src/app/profile/profile.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="topcorner">
+	<mat-accordion>
+		<mat-expansion-panel>
+			<mat-expansion-panel-header>
+				<mat-panel-title>
+					{{ name }}
+				</mat-panel-title>
+			</mat-expansion-panel-header>
+			<mat-card class="example-card">
+				<mat-card-header>
+					<div mat-card-avatar class="example-header-image"></div>
+					<mat-card-title>{{ name }}</mat-card-title>
+					<mat-card-subtitle>{{ description }}</mat-card-subtitle>
+				</mat-card-header>
+				<img
+					mat-card-image
+					src="https://media.npr.org/assets/img/2017/10/27/tra2016_comp_v005_008088.1111_wide-bc335cede10d086ba948db2d66f0093d36f0cc1e-s800-c85.jpg"
```

Comment:
In profile component, we need **imageURL** as an **input** instead of a hard coded URL.